src/bin/tikv-server.rs

```diff
@@ -365,7 +365,9 @@ fn build_raftkv(matches: &Matches,
     let opts = get_rocksdb_option(matches, config);
     let mut db_path = path.clone();
     db_path.push("db");
-    let engine = new_engine_opt(opts, db_path.to_str().unwrap(), DEFAULT_CFS).unwrap();
+    let engine =
+        Arc::new(rocksdb_util::new_engine_opt(opts, db_path.to_str().unwrap(), DEFAULT_CFS)
```

Comment:
There are some places don't need an `Arc<DB>`, for example `EngineRocksdb::new()`.